distribution/ddf-common/src/main/resources-filtered/etc/startup.properties

```diff
@@ -4,19 +4,16 @@ mvn\:ddf.platform.osgi/platform-osgi-condpermadmin/${project.version}=2
 mvn\:ddf.platform.osgi/platform-osgi-conditions/${project.version}=2
 mvn\:org.codice.acdebugger/acdebugger-api/${acdebugger.version}=2
 mvn\:org.codice.acdebugger/acdebugger-backdoor/${acdebugger.version}=2
-mvn\:org.apache.felix/org.apache.felix.metatype/1.2.0=5
+mvn\:org.apache.felix/org.apache.felix.metatype/1.2.2=5
 mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/${karaf.version}=5
 mvn\:org.ops4j.pax.url/pax-url-aether/2.5.4=5
 mvn\:org.fusesource.jansi/jansi/1.17.1 = 8
 mvn\:org.ops4j.pax.logging/pax-logging-api/1.10.1=8
 mvn\:org.ops4j.pax.logging/pax-logging-log4j2/1.10.1=8
 mvn\:org.apache.felix/org.apache.felix.coordinator/1.0.2 = 9
-# Using a boot feature won't be good enough. We need to actually replace Felix with our own impl.
-# The following ddf.platform.osgi line (for the currently used karaf version) used to be:
-# mvn\:org.apache.felix/org.apache.felix.configadmin/1.8.14=10
 mvn\:ddf.platform.osgi/platform-osgi-internal-api/${project.version}=9
 mvn\:ddf.platform.osgi/platform-osgi-configadmin/${project.version}=10
 mvn\:org.apache.felix/org.apache.felix.configadmin/${felix.configadmin.version}=11
```

Comment:
No, technically our osgi bundles are doing exactly what you said. They're plugging into felix's configadmin. I ripped out most of our custom stuff in the last Karaf upgrade. Now we just have our own version of a persistence manager that we're registering in the Felix ecosystem. See this changeset for reference: https://github.com/codice/ddf/pull/3884/files#diff-bb876428036f91da828579112cc86787R247

That being said, I can see how the current naming of our configadmin bundle might be misleading now. Do you think I should rename it?

Reply:
Ah, right. Our stuff needs to start first since we are registering our own persistence manager. No I don't think that's necessary. I think just updating the comment would be sufficient.